File: C/gcd.c
// gcd.c
#include <stdio.h>

int gcd(int a, int b) {
    return (b == 0) ? a : gcd(b, a % b);
}

int main() {
    int a = 56, b = 98;
    printf("GCD of %d and %d: %d\n", a, b, gcd(a, b));
    return 0;
}

File: Java/GCD.java
// GCD.java
public class GCD {
    public static int gcd(int a, int b) {
        return (b == 0) ? a : gcd(b, a % b);
    }

    public static void main(String[] args) {
        System.out.println("GCD of 56 and 98: " + gcd(56, 98));
    }
}

